coreneuron/nrniv/netcvode.cpp

```diff
@@ -393,6 +393,23 @@ PreSyn::PreSyn() {
     output_index_ = 0;
 }
 
+void PreSyn::InitPreSyn() {
+    nc_index_ = 0;
+    nc_cnt_ = 0;
+    flag_ = false;
+    thvar_index_ = -1;
+    pntsrc_ = NULL;
+    threshold_ = 10.;
+    gid_ = -1;
+#if NRNMPI
+    localgid_ = 0;
+#endif
+#if NRN_MULTISEND
+    multisend_index_ = -1;
+#endif
+    output_index_ = 0;
+}
+
```

Comment:
We no longer do cudaMalloc for `nt.presyns`. So this change is not required any more?

Reply:
This shouldn't be necessary anymore since initialization is called on `new`. Do you want me to keep the `InitPresyn()` function anyway or remove everything?

Reply:
I'll remove it. It can be easily recreated if necessary.